### PR TITLE
doc: Update docs for EnableLargeFileShares

### PR DESCRIPTION
**Pull request by nearora-msft**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: docs/driver-parameters.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Name | Meaning | Example | Mandatory | Default value
 --- | --- | --- | --- | ---
 skuName | Azure file storage account type (alias: `storageAccountType`) | `Standard_LRS`, `Standard_ZRS`, `Standard_GRS`, `Standard_RAGRS`, `Premium_LRS` | No | `Standard_LRS` <br><br> Note:  <br> 1. minimum file share size of Premium account type is `100GB`<br> 2.[`ZRS` account type](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy#zone-redundant-storage) is supported in limited regions <br> 3. Premium files shares is currently only available for LRS
 storageAccount | specify Azure storage account name| STORAGE_ACCOUNT_NAME | - No for SMB share </br> - Yes for NFS share|  - For SMB share: if empty, driver will find a suitable storage account that matches `skuName` in the same resource group; if a storage account name is provided, storage account must exist. </br>  - For NFS share, storage account name must be provided
+EnableLargeFileShares | Specify whether to use a storage account with large file shares enabled or not. If this flag is set to true and a storage account with large file shares enabled doesn't exist, a new storage account with large file shares enabled will be created  | `true`,`false` | No | `true`
```
Comment (edreed) [inline docs/driver-parameters.md:11]: Let's be consistent with the rest of the driver parameters and being with a lowercase letter.

File: docs/csi-debug.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ Find out whether you have been throttled by looking at your Azure Monitor by fol
...
+Enable [large file shares](https://docs.microsoft.com/azure/storage/files/storage-files-how-to-create-large-file-share?tabs=azure-portal) on your storage account. Large file shares support up to 10,000 IOPS per share at no extra cost on standard tier. To use a storage account with large file shares enabled, the 'EnableLargeFileShares' driver parameter should be set to true. If this flag is set to true and a storage account with large file shares enabled doesn't exist, a new storage account will be created. The quota needs to be set to 100 TiB to get 10K IOPS. We recommend that you enable large file shares when creating the Storage Account. However, if you do enable this manually for a storage account at a later time, you will not need to remount the file share. Same storage account can have multiple large file shares.
```
Comment (edreed) [inline docs/csi-debug.md:65]: np: lowercase first letter of `enableLargeFileShares`.

File: docs/driver-parameters.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ Name | Meaning | Example | Mandatory | Default value
...
 storageAccount | specify Azure storage account name| STORAGE_ACCOUNT_NAME | - No for SMB share </br> - Yes for NFS share|  - For SMB share: if empty, driver will find a suitable storage account that matches `skuName` in the same resource group; if a storage account name is provided, storage account must exist. </br>  - For NFS share, storage account name must be provided
+enableLargeFileShares | Specify whether to use a storage account with large file shares enabled or not. If this flag is set to true and a storage account with large file shares enabled doesn't exist, a new storage account with large file shares enabled will be created. This flag should be used with the standard sku as the storage accounts created with premium sku have largeFileShares option enabled by default.  | `true`,`false` | No | `true`
```
Comment (andyzhangx) [inline docs/driver-parameters.md:11]: default value is `false`, right?

Comment (nearora-msft) [reply docs/driver-parameters.md]: That's right. I've updated that.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**